content/lua/concepts/mathematical-library/terms/random/random.md

```diff
@@ -0,0 +1,98 @@
+---
+Title: 'random()'
+Description: 'The function generates pseudo-random numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+In Lua, the math.random() function is used to generate pseudo-random numbers. It can be used with or without an argument and in conjunction with the `math.randomseed()` function to control the sequence of random numbers.
```

Comment:
```suggestion
In Lua, the `math.random()` function is used to generate pseudo-random numbers. It can be used with or without an argument and in conjunction with the `math.randomseed()` function to control the sequence of random numbers.
```